src/util.c

```diff
@@ -493,6 +620,17 @@ int string2ll(const char *s, size_t slen, long long *value) {
     return 1;
 }
 
+int string2ll(const char *s, size_t slen, long long *value) {
+#ifdef HAVE_AVX512
+    /* TODO: Cache the cpu info when server startup */
+    if (__builtin_cpu_supports("avx512f") &&
+        __builtin_cpu_supports("avx512vl") &&
+        __builtin_cpu_supports("avx512bw"))
+        return string2llAVX512(s, slen, value);
```

Comment:
are we sure AVX512 outperforms  scalar for any length?

Reply:
I used to change the benchmark commands by using `memtier_benchmark -s 127.0.0.1 -p 9001 --command="LPOS intlist 5"  --hide-histogram --test-time 120`, didn't see an obvious regression for the 1-byte input.

Reply:
Please note that the impact on short lengths, can be invisible on end to end testing, even if its different when measuring the runtime of these functions directly, in this case, we should always run avx512.

Reply:
Branching based on length would be slower?

Reply:
> Branching based on length would be slower?

Yes. In my local environment, using the `slen > 2` check results in a 2% performance regression.